mimesis/builtins/da.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+
+"""Specific data provider for Denmark (da)."""
+
+import random
+
+from mimesis.builtins.base import BaseSpecProvider
+from mimesis.typing import Seed
+
+__all__ = ['DenmarkSpecProvider']
+
+
+class DenmarkSpecProvider(BaseSpecProvider):
+    """Class that provides special data for Denmark (da)."""
+
+    def __init__(self, seed: Seed = None):
+        """Initialize attributes."""
+        super().__init__(locale='da', seed=seed)
+
+    class Meta:
+        """The name of the provider."""
+
+        name = 'denmark_provider'
+
+    def cpr(self) -> str:
+        """Generate a random CPR number (Central Person Registry).
+
+        :return: CPR number.
+
+        :Example:
+            0105865167
+        """
+        day = '{:02d}'.format(random.randint(1, 31))
+        month = '{:02d}'.format(random.randint(1, 12))
+        year = '{:02d}'.format(random.randint(0, 99))
+        serial_number = '{:04d}'.format(random.randint(0, 9999))
+
+        cpr_nr = f'{day}{month}{year}{serial_number}'
```

Comment:
Also, better to use `''.format()` here.

Reply:
Like this? `'{}{}{}{}'.format(day, month, year, serial_number)`

Reply:
@danielarnason Yeah.

Reply:
Ok. Just wanted to make sure. I've gotten so used to using f-strings that I've almost forgotten how to use `''.format()`.